src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -77,6 +78,8 @@ protected override void OnEventCommand(EventCommandEventArgs command)
                 _gen2SizeCounter ??= new PollingCounter("gen-2-size", this, () => GC.GetGenerationSize(2)) { DisplayName = "Gen 2 Size", DisplayUnits = "B" };
                 _lohSizeCounter ??= new PollingCounter("loh-size", this, () => GC.GetGenerationSize(3)) { DisplayName = "LOH Size", DisplayUnits = "B" };
                 _assemblyCounter ??= new PollingCounter("assembly-count", this, () => System.Reflection.Assembly.GetAssemblyCount()) { DisplayName = "Number of Assemblies Loaded" };
+                _ILBytesJittedCounter ??= new IncrementingPollingCounter("il-bytes-jitted-count", this, () => System.Runtime.CompilerServices.RuntimeHelpers.GetILBytesJitted()) { DisplayName = "IL Bytes Jitted", DisplayRateTimeScale = new TimeSpan(0, 0, 1) };
```

Comment:
In the issue Ben Watson mentioned that between the rate and the total count he would prefer the total count. I see here we've got the rate. Not sure if that was accidental or there was a specific reason to do it this way?

Reply:
It was accidental - just fixed it :)